content/chat/getting-started/react.textile

```diff
@@ -0,0 +1,842 @@
+
+* In your @src/app.tsx@ file, create a new component called @PresenceStatus@ like so:
+
+```[react]
+// ChatRoom.tsx
+function PresenceStatus() {
+  // The usePresence hook enters the current client into the room presence
+  usePresence();
+  // The usePresenceListener hook subscribes to presence updates for the room
+  const { presenceData } = usePresenceListener();
+  return (
+    <div className="flex flex-col border-b border-gray-300 bg-white w-full h-full px-4 py-2">
+      <strong className="text-green-700 mr-4 text-center border-b border-gray-900">
+        Online: {presenceData.length}
+      </strong>
+      <div className="flex-1 flex-col flex flex-nowrap items-start gap-4 overflow-x-auto">
+        {presenceData.map((member, idx) => (
+          <div key={idx} className="flex items-center gap-1">
+            <span className="inline-block w-2 h-2 rounded-full bg-green-500" />
+            <span className="text-gray-800">{member.clientId}</span>
+          </div>
+        ))}
+      </div>
+    </div>
+  );
+}
+```
+
+* Then add the @PresenceStatus@ component to your main app component like so:
+
+```[react]
+// App.tsx
+function App() {
+  return (
+  <ChatRoomProvider
+    id="my-first-room" // The room ID you want to create or join
+    release={true} // Release the room automatically when unmounted
+    attach={true} // Attach to the room automatically when mounted
+    options={AllFeaturesEnabled}
+  >
+    <div className="flex flex-col w-[900px] h-full border-1 border-blue-500 rounded-lg overflow-hidden mx-auto font-sans">
+      <div className="flex flex-row w-full border-1 border-blue-500 rounded-lg overflow-hidden mx-auto font-sans">
+        <div className="flex-1 border-1 border-blue-500">
+          <ConnectionStatus />
+        </div>
+        <div className="flex-1 border-1 border-blue-500">
+          <RoomStatus />
+        </div>
+      </div>
+      <div className="flex flex-1 flex-row justify-evenly">
+        <div className="flex flex-col w-1/2 border-1 border-blue-500 rounded-lg overflow-hidden mx-auto font-sans">
+          <div className="flex-1 border-1 border-blue-500 overflow-y-auto">
+          {/* Your PresenceStatus component should go here */}
+            <PresenceStatus />
+          </div>
+        </div>
+        <div className="flex flex-col bg-white w-1/2 border-1 border-blue-500 rounded-lg overflow-hidden mx-auto font-sans">
+          <ChatBox />
+        </div>
+      </div>
+    </div>
+  </ChatRoomProvider>
+);
+}
+```
+
+The component will display a list of clients currently present in the room. The @usePresence()@ hook enters the current client into the room presence, and the @usePresenceListener()@ hook subscribes to presence updates for the room. You should see your current client ID in the list of present users.
```

Comment:
This is a repeat of the opening to this step. Maybe just `You'll now see your current client ID in the list present users`?